kinesis/src/main/scala/akka/stream/alpakka/kinesis/javadsl/KinesisSchedulerSource.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.kinesis.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.NotUsed
+import akka.stream.alpakka.kinesis.{CommittableRecord, scaladsl, _}
+import akka.stream.javadsl.{Flow, Sink, Source, SubSource}
+import software.amazon.kinesis.coordinator.Scheduler
+import software.amazon.kinesis.processor.ShardRecordProcessorFactory
+import software.amazon.kinesis.retrieval.KinesisClientRecord
+
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.Future
+
+object KinesisSchedulerSource {
+
+  abstract class SchedulerBuilder {
+    def build(r: ShardRecordProcessorFactory): Scheduler
+  }
+
+  def create(
+      schedulerBuilder: SchedulerBuilder,
+      settings: KinesisSchedulerSourceSettings
+  ): Source[CommittableRecord, CompletionStage[Scheduler]] =
+    scaladsl.KinesisSchedulerSource
+      .apply(schedulerBuilder.build, settings)
+      .mapMaterializedValue(_.toJava)
+      .asJava
+
+  def create(
+      schedulerBuilder: SchedulerBuilder
+  ): Source[CommittableRecord, CompletionStage[Scheduler]] =
+    create(schedulerBuilder, KinesisSchedulerSourceSettings.defaultInstance)
```

Comment:
Is it worth to have these overloads? Passing the default settings in the examples will do.

Reply:
Seen this pattern elsewhere in Kinesis. Maybe worth to consolidate globally in a later PR.

Reply:
Yes, this is the same as the default arguments in the Scala DSL. I think it is better to make passing the default instance look as nice as possible (maybe call it just `defaults`?).